### PR TITLE
Add ismeuv function

### PR DESCRIPTION
**Pull request by TestSubjector**
*TODO.md: Remove 'ismeuv' from list.
*docs/src/ref.md: Add "ismeuv" entry to the manual.
*src/utils.jl: Add ismeuv entry to "utils".
*src/ismeuv.jl: Contains code of ismeuv function.
*test/util-tests.jl: Include tests for ismeuv.

Additionally, the minexp portion of the code is something which existed in IDL to check underflow with regards to double precision. I don't think it's required, but I'm keeping it until further review,

### CODE REVIEW

**Comment by giordano** [comment]
I didn't look thoroughly to the code, yet, but would it be possible to make the first argument a scalar and take advantage of Julia's syntactic vectorization with dot call? If so, I believe the code would be much simpler.

In IDL there is no such thing like syntactic vectorization, so they often write functions that work for both scalars and vectors.

**Comment by TestSubjector** [comment]
> I didn't look thoroughly to the code, yet, but would it be possible to make the first argument a scalar and take advantage of Julia's syntactic vectorization with dot call?

Good point. I'll try it out.

File: src/ismeuv.jl [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    # minexp = -708.39642  from alog((machar(double=double).xmin)) where double = 1b in IDL
+    z = sqrt(r/(1 - r))
+    denom = -expm1(-2*z*π)
+    tauH = ((3.44e-16)*(r^4)*exp(-4*z*atan(1/z))*hcol)/denom
+    r *= 4
+    tauHe2 = zero(T)
+    if r < 1
+        z = sqrt(r/(1 - r))
+        denom = T(4)
+        denom *= -expm1(-2*z*π)
```
Comment (giordano) [inline src/ismeuv.jl]: You only need one line for this ;-)
```julia
denom = -4 * expm1(- 2 * T(π) * z)
```

File: src/ismeuv.jl [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                          2.500933e-03)
+            if fano
+                episilon = 911.2671/wave
+                for i = 1:4
+                    x = 2*(episilon - esubi[i])/fano_gamma[i]
+                    y += log10(((x - q[i])^2)/(1 + x^2))
+                end
+            end
+        end
+    tauHe1 = exp10(y)*he1col
```
Comment (giordano) [inline src/ismeuv.jl]: While we're at it, indent this line

File: src/ismeuv.jl [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    if r < 1
+        z = sqrt(r/(1 - r))
+        denom = T(4)
+        denom *= -expm1(-2*z*π)
+        tauHe2 = ((3.44e-16)*(r^4)*exp(-4*z*atan(1/z))*he2col)/denom
+    end
+    q  = [2.81, 2.51, 2.45, 2.44]
+    nu = [1.610, 2.795, 3.817, 4.824]
+    fano_gamma = [2.64061e-03, 6.20116e-04, 2.56061e-04, 1.320159e-04]
+    esubi = 4.807317 - (nu.^2).\1
```
Comment (giordano) [inline src/ismeuv.jl]: Looking better, `q`, `fano_gamma`, and `esubi` don't need to be vectors, tuples would work equally good (and are more efficient).  In addition, `esubi` is constant, doesn't depend on input, so in the end you can write
```julia
    q  = (2.81, 2.51, 2.45, 2.44)
    fano_gamma = (2.64061e-03, 6.20116e-04, 2.56061e-04, 1.320159e-04)
    esubi = (4.421529414644497, 4.679309217126802, 4.738680412951545, 4.764345016358231)
```

**Pull request merged**